examples/helpers/ldtk.rs

```diff
@@ -18,13 +23,13 @@ pub struct LdtkPlugin;
 
 impl Plugin for LdtkPlugin {
     fn build(&self, app: &mut App) {
-        app.add_asset::<LdtkMap>()
-            .add_asset_loader(LdtkLoader)
+        app.init_asset::<LdtkMap>()
+            .register_asset_loader(LdtkLoader)
             .add_systems(Update, process_loaded_tile_maps);
     }
 }
 
-#[derive(TypeUuid, TypePath)]
+#[derive(TypeUuid, TypePath, Asset)]
 #[uuid = "abf9eaf2-f21c-4b46-89b0-8aa5c42199af"]
```

Comment:
I think that this and the `TypeUuid` derive are no longer necessary, but you might want to double check that.

Reply:
`TypeUuid` and the `#uuid[...]` portions are no longer needed, but `Asset` is still needed as of testing a moment ago. I'll push changes momentarily.

Reply:
Would you recommend I propagate these changes to other aspects of the code that utilize `TypeUuid` and `Asset` as well?

Reply:
Oops, when I said "this" I meant the `#[uuid` line. Yeah, Asset is fine/needed.

And yeah, I think we can ditch the uuid for the material as well.

Reply:
Okay, done.